scripts/promoted.ts

```diff
@@ -1,4 +1,5 @@
 export const PROMOTED_APP_IDS: string[] = [
     'io.vesea',
-    'com.worldofv.marketplace-nft'
+    'com.worldofv.marketplace-nft',
+    'com.blackvemarket'
```

Comment:
Kindly create a separate PR for this matter. We can discuss and address it there. We primarily consider the transaction volume, user count, and engagement level of the application.